atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -730,6 +732,62 @@ public void testDeleteMultipleVersions() {
     }
 
     @Test
+    public void testDeleteRangeStartRowInclusivity() {
```

Comment:
These look good.  I'm wondering if we should add a test checking for the columns resulting in an error (as per the javadoc), but this would of course involve implementing this protection.  I'm in favor of ensuring each impl has this protect and not just the validator...